### PR TITLE
add license

### PR DESCRIPTION
**Pull request by hroest**
we should add a licence, I suggest BSD

### CODE REVIEW

**Comment by grosenberger** [comment]
We already have the modified BSD licence in ./COPYRIGHT.txt. Is there a better python standard name to store it?

**Comment by hroest** [comment]
true, I didnt see it there - however the file name should be LICENSE see http://lwn.net/Articles/636261/

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+__license__ = """
+Copyright (c) 2012-2014, ETH Zurich, all rights reserved.
```
Comment (grosenberger) [inline LICENSE]: Can we change the copyright to 2012-2015?

**Comment by grosenberger** [comment]
Sounds good. Let's remove the old and add the new one.

**Pull request closed without merging**